packages/design-system/src/components/N8nIconButton/IconButton.vue

```diff
@@ -23,13 +13,15 @@ export default {
 	props: {
 		type: {
 			type: String,
-		},
-		title: {
-			type: String,
+			default: 'primary',
+			validator: (value: string): boolean =>
+				['primary', 'secondary', 'tertiary', 'success', 'warning', 'danger'].includes(value),
 		},
 		size: {
 			type: String,
 			default: 'medium',
+			validator: (value: string): boolean =>
+				['mini', 'small', 'medium', 'large', 'xlarge'].includes(value),
```

Comment:
would recommend removing validation here.. that way `n8n-button` component can handle validation. and we will get one warning instead of 2 warnings.. also don't have to update this one if we change that

Reply:
Good suggestion! Removed.